schulcloud/Controllers/Dashboard/HomeworkOverviewViewController.swift

```diff
@@ -74,6 +74,10 @@ class HomeworkOverviewViewController: UIViewController {
 
 }
 
-extension HomeworkOverviewViewController : ViewControllerHeightDataSource {
+extension HomeworkOverviewViewController : ViewHeightDataSource {
     var height: CGFloat { return 200 }
 }
+
+extension HomeworkOverviewViewController : PermissionInfoDataSource {
+    static var managedPermissions : UserPermissions { return UserPermissions.homeworkView }
```

Comment:
This has to be `requiredPermissions`!

You don't have to use `var` to conform to the protocol. A shorter way to write this would be:
```swift
static let requiredPermissions = UserPermissions.homeworkView
```